src/cloud-element-templates/util/FeelUtil.js

```diff
@@ -39,4 +44,28 @@ export const toFeelExpression = (value, type) => {
   }
 
   return '=' + value.toString();
-};
\ No newline at end of file
+};
+
+export const fromFeelExpression = (value, type) => {
+  if (typeof value === 'undefined') {
+    return value;
+  }
+
+  if (typeof value === 'string' && value.startsWith('=')) {
+    value = value.slice(1);
+  }
+
+  if (type === 'Number') {
+    return Number(value);
```

Comment:
Is this expected to work for `"=100"`? Because `Number("=100")` results in `NaN`.

So for example this:

```
{
  "id": "foobar",
  "type": "Text",
  "value": "Foobar",
  "binding": {
    "name": "bar",
    "type": "zeebe:property"
  },
  "condition": {
    "property": "foo",
    "equals": "=100"
  }
},
```

It's explicitly mentioned here: https://github.com/bpmn-io/bpmn-js-element-templates/issues/195#issuecomment-3606574751